website/src/parsers/scala/index.js

```diff
@@ -0,0 +1,6 @@
+import 'codemirror/mode/clike/clike';
+
+export const id = 'text/x-scala';
```

Comment:
Yeah, it took me a while to figure it out and it's quite ugly as an id.
That said, it seems to be working :)